website/addons/dmptool/static/dmptoolUserConfig.js

```diff
@@ -0,0 +1,231 @@
+/**
+* Module that controls the dmptool user settings. Includes Knockout view-model
+* for syncing data.
+*/
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
references to `knockout.punches` removed: https://github.com/CenterForOpenScience/osf.io/pull/7030/commits/1398fd816b9873fafc40ab3b70593e02be6d0c42